doc/sphinx/source/get-started/installation.rst

```diff
@@ -237,6 +239,196 @@ explained above, if the user has not already done so.
+       rm LHAPDF-6.4.0.tar.gz
+       cd LHAPDF-6.4.0
+
+   Regenerate the configuration files, configure the build with python disabled, compile and 
+   install. You may need to `brew install automake` first:
+
+    .. code::
+
+      autoreconf -f -i
+      ./configure --prefix=$CONDA_PREFIX --disable-python
+      make -j
+      make install
+
+   Install the python wrapper
+
+   .. code::
+
+      cd wrappers/python
+      pip install -e .
+
+   Test
+
+   .. code::
+
+      lhapdf install CT18NNLO
+      python -c "import lhapdf"
+
+6. Apfel
+
+
+   First, we need to install some dependencies:
+
+   .. code::
+      conda install pkg-config swig cmake
+
+   Then build it
+
+   .. code::
+      cd ../../../apfel
+      autoreconf -f -i
+      PYTHON=$(which python) ./configure --prefix=$CONDA_PREFIX 
+      make clean
+      make -j
+      make install
+
+7. validphys
+
+   First install reportengine and validobj, then validphys itself:
+
+   .. code::
+
+      pip install reportengine validobj
+      cd ../nnpdf/validphys2
+      pip install -e .
+
+8. nnpdf
+
+   Install other dependencies
+
+   .. code::
+
+      conda install libarchive sqlite gsl yaml-cpp
+
+   Run cmake in nnpdf/build directory:
+
+   .. code::
+
+      cd ..
+      mkdir build
+      cd build
+      cmake .. -DCMAKE_INSTALL_PREFIX=$CONDA_PREFIX
+
+   Edit the file ``nnpdfgit/nnpdf/CMakeLists.txt`` :
+    - on line 8 change the option to true, so it says:
+
+      .. code::
+
+         SET(CMAKE_BUILD_WITH_INSTALL_RPATH TRUE)
+
+    - comment out line 58 (:code:`set(LIBNNPDF_HAVE_SSE "#define SSE_CONV")`)
+
+    - line 104 should read:
+
+      .. code::
+
+         set(DEFAULT_CXX_OPTIONS "-Wall -Wextra -fvisibility-inlines-hidden -fmessage-length=0 -ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe")
+
+      (so delete ``-march=nocona -mtune=haswell``).
+
+   Then make:
+
+   .. code::
+
+      make -j
+      make install
+
+   Install remaining packages
+
+   .. code::
+
+      pip install seaborn prompt_toolkit scipy psutil hyperopt
+
+9. Install tensorflow
+
+   Not specifying versions will install at the time of writing macos 2.12.0 and metal 0.8.0, which both work.
+   They only give warnings on the optimizers, that the legacy versions are faster.
+   If you want an older version, macos 2.9.2 and metal 0.5.0 are also tested to work.
+
+   .. code::
+
+      conda install -c apple tensorflow-deps
+      pip install tensorflow-macos==2.9.2
+      pip install tensorflow-metal==0.5.0
+
+10. Test
+
+   .. code::
+
+      cd nnpdf/n3fit/runcards/examples
+      vp-setupfit Basic_runcard.yml
+      n3fit Basic_runcard.yml 1
+      evolven3fit Basic_runcard 1
+
+   With these settings tensorflow will run by default on GPU which makes
+   the fit run very slow. To disable the GPU, type the following command:
+   
+   .. code::
+
+      export CUDA_VISIBLE_DEVICES=0
+      
+   or insert the following line in ... [followed by Roy's comment]
```

Comment:
Not sure what my comment was, but I guess it shouldn't be left like this.

Reply:
Yes, it definitely shouldn't be left like this. It should read:

```rst
or insert the following line in in the ``set_initial_state`` function in ``n3fit/src/n3fit/backends/keras_backend/internal_state.py``:

.. code::

      tf.config.set_visible_devices([], 'GPU')
```

And then followed by:
```rst
And to use legacy optimizers, ...
```